### PR TITLE
DEPRECATION notice in README

### PR DESCRIPTION
**Pull request by daks**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

### What type of PR is this?

Deprecation notice in README.

Last commit is from 2013, and there is https://github.com/saltstack-formulas/hostsfile-formula which seems to do the same thing.
Moreover no semantic-release, no doc, no pillar.example.

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Review by javierbertoli — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**DEPRECATED** This formula is not maintained, you should better use https://github.com/saltstack-formulas/hostsfile-formula
```
Comment (myii) [inline README.rst:1]: @daks Do you mind doing this the same way as https://github.com/saltstack-formulas/salt-virt-formula?  We can then archive the formula at the same time.

Comment (daks) [reply README.rst:1]: something like that? (see update)

Comment (myii) [reply README.rst:1]: @daks Not sure why but I don't see the update.  In any case, it's not that important so let's move forward this.  Thanks!

**Review by myii — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (myii) [reply README.rst:1]: @daks I realise what happened!  The original PR was in a `deprecation` branch in _this_ repo and your update was in the `deprecation` branch in _your_ repo!

* https://github.com/daks/hosts-formula/tree/deprecation

I've had this problem in the past as well, it appears to be impossible to rebase a branch in the upstream repo (but you can add more commits).  If you want to use your branch as another PR, we can finalise everything there.

Comment (myii) [reply README.rst:1]: @daks Used your branch in #4 and the formula has been archived.

**Pull request merged**